ktlint/src/test/kotlin/com/pinterest/ktlint/BaseCLITest.kt

```diff
@@ -101,15 +105,17 @@ abstract class BaseCLITest {
                     dir: Path,
                     attrs: BasicFileAttributes,
                 ): FileVisitResult {
-                    Files.createDirectories(dest.resolve(relativize(dir)))
+                    val relativeDir = dir.relativeToOrSelf(this@copyRecursively)
+                    (dest / relativeDir).createDirectories()
```

Comment:
I would prefer to use the `Path.resolve` method instead of `/` operator to avoid confusion with the `/` as path separator.

Reply:
Addressed with adcd6b649d0c96b92ebd3bda66646af17f0aae9a.